app/admin/candidates/components/Actions.js

```diff
@@ -45,6 +63,32 @@ export default function Actions({ launched, slug }) {
     });
     window.location.reload();
   };
+
+  const impersonateUser = async () => {
+    snackbarState.set(() => {
+      return {
+        isOpen: true,
+        message: 'Impersonating user',
+        isError: false,
+      };
+    });
+
+    const impersonateResp = await handleImpersonateUser(email);
+    if (impersonateResp === true) {
+      let candidatePage = `/candidate/${slug}`;
+      await revalidatePage(candidatePage);
+      window.location = candidatePage;
```

Comment:
window.location.href or window.location.redirect, how did this work for you locally?

```
// Simulate a mouse click:
window.location.href = "httpa://www.google.com";

// Simulate an HTTP redirect:
window.location.replace("https://www.google.com");

```

Reply:
this is still an issue @taylorgp